### DISCUSSION TITLE
Limit the Feed to the Most Upvoted

### DISCUSSION THREAD

**Discussion by askpatrickw** [dailydotdev/daily#1412] [Feature Requests]
Who has time to read all this... I'd rather see the top posts and filter out the rest. I'd like to see nothing if I've already seen all the top posts.

Stop the scrolling treadmill.

**Comment by idoshamun** [comment]
Hi @askpatrickw,

I appreciate your opinion, but many other users actually would prefer fresh content or the feed as currently designed.
This is exactly why we have several feeds so anyone can find the best one for them, feel free to use only the most upvoted one if this works for you
